torch/export/_trace.py

```diff
@@ -1156,6 +1156,7 @@ def forward(self, *args, **kwargs):
     _normalize_nn_module_stack(gm_torch_level, type(mod))
 
     # NOTE: graph module expects only positional args
+    constant_attrs = _gather_constant_attrs(mod)
```

Comment:
Remove this one, this _export_nont_strict will deal with this after the movement mentioned in previous comment is done.